R/dsl.R

```diff
@@ -7,3 +7,29 @@
 uiicon <- function(type = "", ...) {
   tags$i(class = paste(type, "icon"), ...)
 }
+
+#' This creates a default dropdown using Semantic UI styles with Shiny input. Dropdown is already initialized
```

Comment:
This isn't displayed correctly. We need to add extra line above the description with function name.